<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstructorDialogExtended</class>
 <widget class="QDialog" name="ConstructorDialogExtended">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="edtReportName"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnCreateReport">
     <property name="text">
      <string>CreateReport</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Шапка таблицы</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="2">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Формат ячейки</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>Обычный</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Жирный</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Курсив</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Стиль</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lblAlignmentCapItem">
               <property name="text">
                <string>Выравнивание</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cmbAlignmentCapItem">
               <item>
                <property name="text">
                 <string>Слева</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>По центру</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Справа</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="2">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="2" column="2">
             <widget class="QSpinBox" name="spinBoxColumnCount">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QPushButton" name="btnCreateTableCap">
              <property name="text">
               <string>Создать таблицу</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QSpinBox" name="spinBoxRowCount">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnClearTableCap">
            <property name="text">
             <string>Очистить таблицу</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="btnFillTableCap">
            <property name="text">
             <string>Заполнить поля</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btnDeleteTableCap">
            <property name="text">
             <string>Удалить таблицу</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="6">
           <widget class="CRCTableCapView" name="tblCap">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Параметры</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="CInDocTableView" name="tableParams"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxSQL">
         <property name="title">
          <string>Текст Запроса</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="3">
           <widget class="QTextBrowser" name="textSql">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnApplySql">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnResetSql">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library.InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CRCTableCapView</class>
   <extends>QTableView</extends>
   <header>Reports/ReportsConstructor/RCTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConstructorDialogExtended</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConstructorDialogExtended</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
